### ISSUE TITLE
Does multiformat support render API as well?

### ISSUE THREAD

**Issue by yangling1985** [prebid/prebid-mobile-android#621]
Hi, team, 

I notice that multifomat is supported for original API: https://github.com/prebid/prebid-mobile-android/pull/610. Does it support or will support render API as well? thanks!

**Comment by YuriyVelichkoPI** [comment]
Hi @yangling1985 ! Multiformat is supported for Interstitial ad units in Rendering API. 

Do you need the support of multiformat banner ad units as well?

**Comment by YuriyVelichkoPI** [comment]
Hi @yangling1985! Sorry for the late response. 

You can find how to start multiformat bidding on each doc page for the rendering API. For example on the [AdMob](https://docs.prebid.org/prebid-mobile/modules/rendering/android-sdk-integration-admob.html#interstitial-api). 

In order to make a multiformat bid request, set the respective values into the adUnitFormats parameter.

```
adUnit = MediationInterstitialAdUnit(
            activity,
            configId,
            EnumSet.of(AdUnitFormat.BANNER, AdUnitFormat.VIDEO),
            mediationUtils
        ) 
```

**Comment by yangling1985** [comment]
np. thanks! @YuriyVelichkoPI

**Comment by yangling1985** [comment]
Hi @YuriyVelichkoPI, for multiformat,  does prebid now supports banner competes with native: something like below?
```
adUnit = BanerAdUnit(
            activity,
            configId,
            EnumSet.of(AdUnitFormat.BANNER, AdUnitFormat.NATIVE),
            mediationUtils
        ) 
```
I notice doc says: "Prebid multi-format ad units allow direct competition between **banner**, **native**, and/or video bids: " https://docs.prebid.org/dev-docs/show-multi-format-ads.html#how-multi-format-ads-work

**Comment by YuriyVelichkoPI** [comment]
Hi @yangling1985 ! No Prebid SDK does not support NATIVE for multiformat bidding in any API. 

The doc that you shared relates to `Prebid.js`. 

I can bump this request on the following Prebid Mobile committee so the team can prioritize implementation. Could you provide the following info to accelerate prioritization: 
- which company do you represent?
- do you have resources to contribute to the development of this feature (iOS, Android engineers)?

If you are already a part of the committee, feel free to raise an issue by yourself. 

cc: @mmulin

**Comment by yangling1985** [comment]
Hi, @YuriyVelichkoPI, 
I am from NewsBreak and already a part of the committee. About the resources to contribute to the development of this feature, do you have an rough estimation of the effort(like how many man day or man month), for Android and iOS separately?

**Comment by YuriyVelichkoPI** [comment]
> I am from NewsBreak and am already a part of the committee. 

Great! Looking forward to helping you with prebid integration. 

> do you have an rough estimation of the effort

I don't have an ETA right now, will prepare it early next week. 

Few more questions to clarify requirements:
- which Ad Server do you use, or wich one are you going to use? I'm pretty sure that it is GAM, but I want to be sure. 
- are you sure you need to integrate Rendering API  but not an Original API? If you need deeper consultation about the purpose and difference between Rendering API / Original API, let me know, and we'll arrange a workshop to clarify all peculiarities.

**Comment by yangling1985** [comment]
yes. we are using GAM. But we prefer to fetch ads using GAM SDK built-in API for now and use Prebid SDK for other demands. in another way, we want to do client-side mediation on our own. 
In the future, we want put GAM as a prebid bid partner. actually we are pushing Google to support ORTB server-to-server integration. hopes that answered your questions. 

Again, we are really looking forward to see the multi-format supports native for the Prebid mobile SDK so that we can just use one single ad unit(in-hence single one bid request) to support both native and banner ads.  thanks in advance! @YuriyVelichkoPI

**Comment by YuriyVelichkoPI** [comment]
Thanks for the explanation @yangling1985 ! It will help to build a proper solution. 

Support for such behavior is viable. But I need more details to design the solution and provide proper estimates.

Which kind of implementation of native ads do you use and need:
- [Native Styles] (https://developers.google.com/ad-manager/mobile-ads-sdk/android/native/styles) rendering, when the native ad is rendered via HTML markup and displayed in the WebView 
- [In-app layout](https://developers.google.com/ad-manager/mobile-ads-sdk/android/native/advanced) with native iOS/Android views
- Both :) ?

Also, do you need support for video assets?

**Comment by YuriyVelichkoPI** [comment]
cc: @alexsavelyev for viewability

**Comment by yangling1985** [comment]
Thanks. @YuriyVelichkoPI. we are implementing native ads in the **In-app layout** way only.

**Comment by YuriyVelichkoPI** [comment]
Hi @yangling1985 ! What about video? Do you use native ads with video?

**Comment by yangling1985** [comment]
@jsligh thanks for checking. we are good now with the original API.  please feel free to close this ticket.

**Comment by YuriyVelichkoPI** [comment]
SDK doesn't fully support multiformat in Rendering API yet.
